contrib/clojure-package/project.clj

```diff
@@ -36,8 +36,8 @@
                  [org.apache.logging.log4j/log4j-api "2.8.1"]
                  [org.slf4j/slf4j-log4j12 "1.7.25" :exclusions [org.slf4j/slf4j-api]]]
 
-                 [org.scala-lang/scala-library/"2.11.8"]
-                 [org.scala-lang/scala-reflect/"2.11.8"]
+                 [org.scala-lang/scala-library "2.11.8"]
+                 [org.scala-lang/scala-reflect "2.11.8"]
```

Comment:
The error on the CI is because there is a missing `]` in the formatting. The ending bracket on line 37 needs to be moved to line 40

Reply:
My bad.  This happens when patching files you don't understand.  Thanks @gigasquid